articles/intro-to-C++-part01/index.md

```diff
@@ -0,0 +1,134 @@
+4. int main() {
+5. cout<<"Hello World"<<endl;
+6. return 0;
+7. }
+
+
+This program will print Hello World to the screen. Let's examine it line by line to see what it means.
+
+- Line 1 - **//This is a sample C++ Program**
+
+Any line of a program that begins with a "//" is a comment. The compiler will ignore this line.
+
+Comments are there to be read by people. You should always write comments for each major step of the algorithm to explain something weird or cool that is happening. That way, anyone working along with you or for you would be able to understand what you were trying to do.
+
+- Line 2 - **#include<iostream>**
+
+C++, by itself, cannot do input or output, so we add functionality to C++ by including libraries. For example, :<iostream> is a library that has commands for input from the keyboard and output to the screen.
+
+- Line 3 - **using namespace std;**
+
+It is something that you cannot avoid and have to write in every single program that you will write in your whole life.
+
+- Line 4 - **int main ( ) {**
+
+The aforementioned tells the compiler that this is the beginning of the main program. **{** means the start of the block of the program.
+
+- Line 5 - **cout<<"Hello World"<<endl;**
+
+**cout** is our "output to the screen" statement. **"endl"** takes you the next line. **";"** ends a C++ statement.
+
+- Line 6 - **return 0**
+
+This means that this is the end of our program.
+
+- Line 7 - **}**
+
+The line above is our closing bracket to end the program.
+
+Congratulations! You just wrote your first C++ program.
+
+Let's write another program that would read two numbers and output the sum of them.
+`  // program to add two numbers
+
+     #include <iostream>
+     using namespace std;
+     int main(){
+     cout<<"enter a number"<<endl;
+     int num_one;
+     cin>>num_one;
+     cout<<" Enter another number"<<endl;
+     int num_two;
+    cin>>num_two;
+    int sum;
+    sum = num_one + num_two;
+    cout<<num_one<<"+"<<num_two<<"="<<sum<<endl;
+      return0;
+    }`  
+
+Let's discuss the things that are new to us in this program above.
+
+1. **int num_one** - This means that we are reserving space in memory to hold an integer (int). And we are naming that space num_one.
+
+Integers hold positive as well as negative values (and 0) and include non-decimal numbers up to about +-2 billion, which gives you an exact value between +- 2 billion.
+
+Just like int, we also have double. It can hold large as well as small numbers. It can also be an exponent. Here is how we declare it.
+
+`double first_one;`   
+
+2. **cin>>num_one** - This is the way of accepting an input from the user in the same memory location that we reserved earlier.
+
+And that is all you need to know to write basic programs in C++. We will go over other new things as we continue.
+
+Why don't you work on some practice problems before moving forward? For example you could try to:
+
+- Write a program to convert a temperature from Fahrenheit to Celsius?
```

Comment:
Placing myself in the audience Im left wanting more but not in a good way - Im left as if I need further explanation or clarification.